native/src/components/EventListItem.tsx

```diff
@@ -13,6 +13,7 @@ const Description = styled.Text`
   font-family: ${props => props.theme.fonts.native.contentFontRegular};
 `
 
+const NUM_OF_CHARS_ALLOWED = 90
```

Comment:
I feel like we should make this the default for native and therefore put it in a constant somewhere. But I can do that in https://issues.tuerantuer.org/browse/IGAPP-1086 as well.

Reply:
Your choice!

Reply:
I will do it :)